content/body/text-resize.php

```diff
@@ -229,38 +228,30 @@ functionality
   <li><strong>Desktop:</strong>
     <ul>
       <li>At the top right, click More <span aria-hidden="true">⋮</span> and then Settings.</li>
-      <li>Under "Appearance," nex" to "Font size," click the Down arrow <span aria-hidden="true">▾</span>.
-        Then select the font size you want (you have a choice
-        of
-        very small, small, medium, large and very large). You can have a little bit more
-        granular
-        control by clicking "Customize fonts" and moving the "Font Size" range widget.</li>
+      <li>Under "Appearance," next" to "Font size," click the Down arrow <span aria-hidden="true">▾</span>.
+          Then select the font size you want (you have a choice of very small, small, medium, large, and very large). 
+          You can have a little bit more granular control by clicking "Customize fonts" and moving the "Font Size" range widget.</li>
     </ul>
     <p><em>Note that Chrome will <strong>not</strong> resize text that is sized in <code>px</code>
         units.</em></p>
   </li>
   <li><strong>Mobile (Android):</strong>
     <ul>
-      <li>Go to Settings, and then Accessibility. You can change the font-size by using the "Text
-        Scaling" slider.</li>
+      <li>Go to Settings, and then Accessibility. 
+          You can change the font size by using the "Text Scaling" slider.</li>
     </ul>
-    <p><strong><em>Please note that Chrome for Android has some serious differences than all other
-          browsers. Text is only resized inside HTML element has more than 217 characters in
-          it,
-          and only if they have a dynamic height . This is not useful as an accessibility
-          feature,
-          since it is not guaranteed to resize all the content on the page. Because of this,
-          text-zoom-resize does not support Chrome for Android.</em></strong></p>
-    <p><a href="https://bugs.chromium.org/p/chromium/issues/detail?id=779409">A bug has been filed a
+    <p><strong><em>Please note that Chrome for Android has some serious differences from all other browsers. 
+      Text is only resized inside HTML elements has more than 217 characters in it, and only if they have a dynamic height. 
```

Comment:
I think the following change makes more sense:
```suggestion
      Text is only resized inside HTML elements with more than 217 characters in it, and only if they have a dynamic height. 
```